javascript2-week1/week1.html

```diff
@@ -0,0 +1,52 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quiz App Prototype</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="quiz-container">
+        <form id="quiz-form">
+            <label for="question">Question (max 140 chars):</label>
+            <input type="text" id="question" maxlength="140" required>
+            <div class="answers">
+            <div>
+                <input type="radio" name="correct-answer" id="answer1" required>
```

Comment:
The homework tasks mentions about 4 possible anwsers and you have 7 on your page, I think you can remove first 3 :)
Because of that you also have duplicated id's, wich may cause different problems, with both validation and accessibility of your form.